pkg/ddl/multi_schema_change.go

```diff
@@ -55,6 +65,16 @@ func (d *ddl) MultiSchemaChange(ctx sessionctx.Context, ti ast.Ident) error {
 	return d.callHookOnChanged(job, err)
 }
 
+func containsDistTaskSubJob(subJobs []*model.SubJob) bool {
+	for _, sub := range subJobs {
+		if sub.Type == model.ActionAddIndex ||
```

Comment:
Once there is add index or add pk then we make it distributed sub job? this means the multi schema change support part subjob be executed on dist framework?

Reply:
Yes.